Sequence of Tenses/maxi.md

```diff
@@ -0,0 +1,39 @@
+# Please translate the following sentences into English
+
+1. На прошлой неделе Сергей заметил, что его коллега по проекту пользуется готовыми снипетами, что ускоряло его работу, но видимо он передумал и решил писать код по старинке.
+
+On the previous week, Sergiy noticed, that his colleague on the project using the prepared snippet of code, that increased the speed of work, but he changed the mind and decided writing the code as usual 
+
+2. Тим лид признался что всем придется писать юнит тесты по своему функционалу и то что они делали до этого придется переделать с ноля, так как отдел контроля качества забраковал первичный вариант проекта.
+
+Team lead admitted that everyone must write the unit tests for functionality that they did,  and everything that previously had done need to be reworked from scratch, because QA discards the very first variant of the project. 
```

Comment:
would have to write unut tests - future in the past as the main clause is in the past and regarding unit tests they are used in plural and we have combo thus we should use zero article unless you mean special unit tests
the functionality  - definite article cause we precise the information later
everything that had been previously done - Past Perfect Passive (without Passive - everything that they had been doing previously
becasue QA had discarded